### PR TITLE
 Quiver Tab "Projects" selected although Tab "Workflow" is opened

### LINKED ISSUE #57: Quiver Tab "Projects" selected although Tab "Workflow" is opened
When opening the URL https://ocr-d.de/quiver-frontend/#/workflows, the tab "projects" is highlighted although the page "workflow" is opened.
This case also applies when opening https://ocr-d.de/quiver-frontend/#/processors.

Task: Highlight the associated tab.

Image

### PR DESCRIPTION
**Pull request by noornoorie**
Closes #57

### CODE REVIEW

File: src/views/HomeView.vue [hunk 1]
```diff
@@ -25,14 +26,21 @@ const items = ref([
   }
 ])
 
-function onTabChange({ index }) {
-  router.push(items.value[index].to)
-}
+const activeRouteIndex = computed({
```
Comment (paulpestov) [inline src/views/HomeView.vue:29]: We also have to consider that the URL stays the same which shouldn't be the case. I think the solution should be to redirect the route to /workflows before loading the tabs. Then the tabs would pick up the right path automatically.

Comment (noornoorie) [reply src/views/HomeView.vue:29]: I'm not really sure about the issue, could you please elaborate further?

Comment (paulpestov) [reply src/views/HomeView.vue:29]: When you visit the app initially it stays on `/`. But in fact it is `/workflows`, so it should redirect the URL to `/workflows`. After that you should pass the index to the TabMenu. So the getter here becomes simpler. That way the tabs would always stay in sync with the  URL.

Comment (noornoorie) [reply src/views/HomeView.vue:29]: Okay, I've updated it. Can you check it out?

**Comment by noornoorie** [comment]
@paulpestov Can you check this pull request and merge if there's no issues? I still don't have write access to the repository

**Pull request merged**